spec/models/email_spec.rb

```diff
@@ -72,7 +72,7 @@
       it 'includes format' do
         expect(document).to match('Format: Senior Thesis')
       end
-      it 'includes library name when location field is not present in holdings' do
+      it 'includes only library name when location field is not present in holdings' do
```

Comment:
@XanderGardner can you please update this test as well? I'm sorry I missed it earlier. If you'd like to check the record:  https://catalog.princeton.edu/catalog/dsp01zk51vk08g/raw. It doesn't have holdings_1display. You can remove it. You will have to adjust your test and remove the last one `..to match('Location: Online')`.

Reply:
@XanderGardner yes if you can update this. It would be great.